### ISSUE TITLE
Bug: Transformer\MinifyHtml breaks cyrillic letter 'х' followed by whitespace

### ISSUE THREAD

**Issue by Likhachev** [ampproject/amp-toolbox-php#420]
```php
$unoptimizedHtml = "
А а	Б б	В в	Г г	Д д	Е е	Ё ё
Ж ж	З з	И и	Й й	К к	Л л	М м
Н н	О о	П п	Р р	С с	Т т	У у
Ф ф	Х х	Ц ц	Ч ч	Ш ш	Щ щ	Ъ ъ
Ы ы	Ь ь	Э э	Ю ю	Я я";

$transformationEngine = new TransformationEngine();
$errorCollection      = new ErrorCollection; 

$optimizedHtml = $transformationEngine->optimizeHtml(
    $unoptimizedHtml, 
    $errorCollection 
);
```

Output

```
А а	Б б	В в	Г г	Д д	Е е	Ё ё
Ж ж	З з	И и	Й й	К к	Л л	М м
Н н	О о	П п	Р р	С с	Т т	У у
Ф ф	Х � Ц ц	Ч ч	Ш ш	Щ щ	Ъ ъ
Ы ы	Ь ь	Э э	Ю ю	Я я
```

Works only with space right after the letter:

`ххххх ` => `хххх�`
but
`ххххха ` => `ххххха `

The problem in `\AmpProject\Optimizer\Transformer\MinifyHtml::normalizeWhitespace`

Temporary workaround - turn off whitespace collapsing

```php
$configurationData = [
    Transformer\MinifyHtml::class => [
        Configuration\MinifyHtmlConfiguration::COLLAPSE_WHITESPACE => false,
    ],
];
```

**Comment by ediamin** [comment]
@Likhachev Thanks for reporting the bug. I've found that whitespace around Cyrillic `х` has causes issue when we try to use the vertical tab `\v` in the [preg_replace function](https://github.com/ampproject/amp-toolbox-php/blob/main/src/Optimizer/Transformer/MinifyHtml.php#L239). I'll post an update with possible fix.

**Comment by swissspidy** [comment]
Seeing similar issues with Persian strings like `داروی جامع امام رضا + کاربرد و نحوه مصرف دارو`:

![Screenshot 2021-12-01 at 21 16 35](https://user-images.githubusercontent.com/841956/144307334-e3e3d3f8-e907-472b-9ff2-afe294b411d8.png)

Appearance is fine again when not using the `MinifyHtml` transformer.

**Comment by schlessera** [comment]
This issue should have been solved with #421. @swissspidy can you confirm by testing with 0.9.0+?

**Comment by swissspidy** [comment]
Can confirm that this is now fixed!

**Comment by schlessera** [comment]
Fixed via #421
